README.md

```diff
@@ -68,3 +68,30 @@ This command is used to build a Docker image from a Dockerfile. Here's what each
 - `.`: This specifies the build context, which is the location of the Dockerfile and any files that are being copied into the Docker image. In this case, it's the current directory.
 
 So, this command builds a Docker image for Linux systems with AMD64 architecture, tags it with the name `qsofte` and the version `v0.10`, and uses the current directory as the build context.
+
+## Setup Local Dev Environment
+
+1. clone the repository remote to local using command 'git clone link of repository'
+
+2. switch into the repository file using command 'cd fileName'
+
+3. install npm using command 'npm install'
+
+4. create new branch using command 'git checkout -b "branchName" '
+
+5. make some changes into repository .
```

Comment:
Make the required changes as per user story

Reply:
There is not need to write details of git changes all we want is that on local machine this project should start. 

Add npm install etc 

@Mayank-Qsofte - are you able to peer review the documents as well please ?

Reply:
yes doing so right now